metal3-dev/local-cbo.sh

```diff
@@ -18,6 +33,11 @@ fi
 cd $cbo_path
 
 export RUN_NAMESPACE=openshift-machine-api
-oc apply -f manifests/
+oc apply -f manifests/0000_31_cluster-baremetal-operator_01_images.configmap.yaml
+oc apply -f manifests/0000_31_cluster-baremetal-operator_02_metal3provisioning.crd.yaml
+oc apply -f manifests/0000_31_cluster-baremetal-operator_03_baremetalhost.crd.yaml
```

Comment:
This CRD needs to match the version used by the BMO in the cluster. If we're not changing the BMO image in this script, we should keep the CRD that has already been installed when the cluster was built. If we're running a local BMO, the script to do that will update the CRD itself.